requirements.txt

```
@@ -29,7 +29,7 @@ django-picklefield==2.1.1
 django-pipeline==1.7.0
 django-q==1.1.0
 django-recaptcha==2.0.6
-djangorestframework==3.11.0
+djangorestframework==3.10.0
```

Let's try the latest point release of 3.10

```suggestion
djangorestframework==3.10.3
```